contrib/online_ide/sources/nit/pnacl_nit.nit

```diff
@@ -82,7 +82,8 @@ redef class FileReader
 
 	redef fun fill_buffer
 	do
-		_buffer.clear
+		_buffer_pos = 0
+		_buffer_length = 0
```

Comment:
Yup, but since the _buffer is now a NativeString, there's no clear method on that :)